src/r-apt/devcontainer-feature.json

```diff
@@ -54,7 +54,7 @@
 		"installBspm": {
 			"type": "boolean",
 			"default": false,
-			"description": "Install and enable BSPM (Bridge to System Package Manager) to install R packages."
+			"description": "Install and enable BSPM (Bridge to System Package Manager) to install R packages. This option is only working on Ubuntu now."
```

Comment:
Good qualifier. We should also add '22.04' or 'jammy'.

Reply:
Oops, is r2u not currently building for focal?

Reply:
It does. And is used. But jammy is simply better so why cling to focal too?  (I edited my comment after the first pass too as I initiallt worded it confusingly, my bad).

Reply:
As I mentioned here, I think focal support is important to satisfy the demand to add R to the universal image since the universal image is focal-based.
https://github.com/rocker-org/devcontainer-features/pull/169#discussion_r1285065939